### DISCUSSION TITLE
Athens Newsletter

### DISCUSSION THREAD

**Discussion by jsmorabito** [athensresearch/athens#1355] [Operations]
The State of Athens newsletter
### purpose
update the community with the following information:
- product updates
- development progress
- community highlights
  - links to community calls
- featured contributors or contributions
- upcoming events
- Athens tips, tricks, workflows
  - user from community can share their use case and how theyre using athens
- newly posted or high priority Gitcoin bounties

provide project information:
- links/invitations to github, youtube, twitter, discord, handbook
- contact email for support
- link to hiring

### current state
- a newsletter is not currently being sent out
- we have a ghost blog which can be used to send out the newsletter
- e-mail collection strategy is not clear at the moment, will be tied to the website
- proposing state of athens discussion until we have a better collaborative method (see #1357) 
  - thinking that we can have a discussion post to collect ideas for the newsletter then post it through ghost at the end of the month

### questions
- what degree of involvement can the community have in the making of the newsletter and what does that look like?
- should a community driven update be a separate entity which is then highlighted in a project maintainer run newsletter?

### notes
- @sawhney17 created an excellent example for the month of May 2021 which we can build off of: https://www.notion.so/What-s-new-in-Athens-da66d490d9f84abb8bf4f0732362be07
- newsletter can be entry point to contributor pipeline
  - welcome new learners in #1281 , highlight first time contributors
- newsletter inspiration
  - Gitcoin
  - Clover
  - Roam Book Club

**Comment by sawhney17** [comment]
### What kind of language/tone should be followed?
Should we go informal or more on the formal side of things? In my May example, I used an informal tone but this is something really dependent on what is the target of the newsletter. Is it meant to be friendly and invite everyone to keep up with the development or is it targeted at people who are just actively interested in the development. I'm subbed to this guy James Smith PT. His newsletter is about the most 

### What could be a frequency of the newsletter?
To me, I think monthly is a bit too low of a frequency, but at the same time, with an Athens newspaper, we aren't really providing them the value that you would expect from weekly ones from guys like Nat Eliason. I think twice a month strikes a nice balance and one could be like the State of Athens and the other could be more of a community driven one less focused on what was accomplished and more on using the tool and what is coming soon. Maybe the state of athens will be formal and the other more informal?

**Reply by joelhans** [reply]
I actually see your **What's new in Athens** document as being a separate piece of content from this newsletter. This document would be recurring on a regular basis, and would partially inform an email newsletter, but wouldn't be it's entirety.

I may create a separate discussion thread around how to handle the creation of this document, or maybe use #1357 to brainstorm some about how to create this content and what regular processes we should follow to make sure we're actually following through on writing this document at the frequency we decide.

**Comment by joelhans** [comment]
My big inspiration newsletter-wise lately has been [The Observatory](https://us4.campaign-archive.com/?u=1344d782b1c544a021cd5a360&id=0671f7ad9f). It's informative, narrative, but also highly engaging. It provides intellectual value to readers while also leaving room for some of the more formal updates mentioned in the parent comment, like upcoming events or community highlights.
